src/web_interface/templates/show_statistic.html

```diff
@@ -6,7 +6,7 @@
 {% set count = [] %}
 
 {% block head %}
-<script src="{{ url_for("static", filename="Chart.js") }}"></script>
+<script src="{{ url_for("static", filename="node_modules/chart.js/dist/chart.js") }}"></script>
```

Comment:
It seems it should be `"Chart.js"` again with the older version.
```suggestion
<script src="{{ url_for("static", filename="node_modules/chart.js/dist/Chart.js") }}"></script>
```
Searching seems to work again then

Reply:
The `TypeError: Bootstrap's tooltips require Popper.js` error seems to vanish when you use `popper.js/dist/umd/popper.min.js` instead of `popper.js/dist/popper.min.js` (I'm not sure why, though).

Reply:
The progress bar seems to be still broken. But according to the Bootstrap docs we're doing it wrong anyways (that is probably still Bootstrap 3.x syntax). I have created a small patch that should fix this (and always display the text at the center of the bar instead of the center of the larger part):

```diff
diff --git a/src/web_interface/static/js/system_health.js b/src/web_interface/static/js/system_health.js
index 949f8eb5..9fe9b85b 100644
--- a/src/web_interface/static/js/system_health.js
+++ b/src/web_interface/static/js/system_health.js
@@ -53,9 +53,10 @@ function getProgressBar(percentage, labelCurrent, labelMax, unit) {
     const value = `${labelCurrent} ${unit} / ${labelMax} ${unit}`;
     return `
         <div class="progress-bar text-center${percentage > 80 ? " bg-warning" : ""}" role="progressbar" style="width: ${percentage}%">
-            ${percentage >= 50 ? value : ""}
         </div>
-        &nbsp;&nbsp;${percentage < 50 ? value : ""}
+        <div class="justify-content-center d-flex position-absolute w-100" style="margin-top: 10px;">
+            ${value}
+        </div>
     `;
 }
 
diff --git a/src/web_interface/templates/system_health.html b/src/web_interface/templates/system_health.html
index 2276331a..a648386e 100644
--- a/src/web_interface/templates/system_health.html
+++ b/src/web_interface/templates/system_health.html
@@ -46,13 +46,13 @@
                 <tr>
                     {{ icon_tooltip_desk('memory', 'Memory status') }}
                     <td colspan="5" class="align-middle">
-                        <div class="progress" style="height: 20px;" id="{{ component }}-memory"></div>
+                        <div class="progress position-relative" style="height: 20px;" id="{{ component }}-memory"></div>
                     </td>
                 </tr>
                 <tr>
                     {{ icon_tooltip_desk('hdd', 'Disk status') }}
                     <td colspan="5" class="align-middle">
-                        <div class="progress" style="height: 20px;" id="{{ component }}-disk"></div>
+                        <div class="progress position-relative" style="height: 20px;" id="{{ component }}-disk"></div>
                     </td>
                 </tr>
                 {% if component == "backend" %}

```

Reply:
I applied your patch and fixed the chart.js path.